dataretrieval/nwis.py

```diff
@@ -126,12 +126,14 @@ def get_qwdata(
 
     .. warning::
 
-        WARNING: Beginning in February 2024 the NWIS qw data endpoint will not deliver new data or updates to existing data.
+        WARNING: Beginning in late February 2024 the NWIS qw data endpoint will
```

Comment:
looks like this one managed to evade the date update, it should be March 2024.

Reply:
Nice catch, thanks! Changed.